src/Task/Git/CommitMessage.php

```diff
@@ -96,7 +96,18 @@ public function run(ContextInterface $context)
         }
 
         if (count($exceptions)) {
-            return TaskResult::createFailed($this, $context, implode(PHP_EOL, $exceptions));
+            return TaskResult::createFailed(
+                $this,
+                $context,
+                implode(PHP_EOL, $exceptions) . <<<KAWAii
+</>
```

Comment:
This makes the commit message visually distinguishable from the error output since it is not really part of the error. Furthermore, the overuse of red becomes hard to read at some point.